ISM43362/ISM43362.cpp

```diff
@@ -83,7 +83,7 @@ const char *ISM43362::get_firmware_version(void)
     char tmp_buffer[250];
     char *ptr, *ptr2;
 
-    if(!(_parser.send("I?") && _parser.recv("%s\r\n", tmp_buffer) && check_response())) {
+    if(!(_parser.send("I?") && _parser.recv("%[^\n]\r\n", tmp_buffer) && check_response())) {
```

Comment:
can you mention what [^\n]  refers to ?  
Also please update commit title so that the title can fit in one line.

Reply:
Ok to add a comment and rephrase the commit, done.